Makefile

```diff
@@ -92,6 +94,47 @@ installcheck:
 	echo $$V && test `echo "$$V" | awk '{print $$1}'` = "$(VERSION)"
 	dropdb linz-bde-schema-test-db
 
+# TODO: run the full test after preparing the db ?
+check-loader-stdout:
+
+	dropdb --if-exists linz-bde-schema-test-db
+
+	createdb linz-bde-schema-test-db
+	linz-bde-schema-load - | \
+        psql -Xo /dev/null linz-bde-schema-test-db
+	linz-bde-schema-load - | \
```

Comment:
Duplicated line?

Reply:
It is actually intentional, we test that the script is idempotent, basically.
I'd like to reduce the overall duplication though, one day...